main/src/modules/jointsAwareness/include/jointsAwareness.h

```diff
@@ -0,0 +1,88 @@
+#include <iostream>
+#include <sstream>
+#include <map>
+#include <fstream>
+#include <string>
+#include <map>
+#include <yarp/dev/Drivers.h>
+#include <yarp/dev/CartesianControl.h>
+#include <yarp/dev/GazeControl.h>
+#include <yarp/dev/PolyDriver.h>
+#include "wrdac/clients/icubClient.h"
+
+
+
+using namespace wysiwyd::wrdac;
```

Comment:
Sure :) I was not using them anyway that was a copy/paste from somewhere that I forgot to delete